test/k8s-e2e/convtojson.py

```diff
@@ -72,7 +74,7 @@ def convert_junit_01_xml_to_json(filename, index, targetdir, cloudprovider, exec
     return 0
 
 
-HELPMESSAGE = 'convtojson.py -i|--inputfile <xml inputfile> --index <elastic search index> -c|--cloudprovider -d|--targetdir <directory for json files> -t|--timestamp'
+HELPMESSAGE = 'convtojson.py -i|--inputfile <xml inputfile> --index <elastic search index> -c|--cloudprovider -d|--targetdir <directory for json files> -t|--timestamp -f|--descfile <test description file>'
```

Comment:
Maybe we should rename inputfile and descfile so that we know what we actually want for these flags